Subsequence_Bitmasking/subsequence_bitmasking.cpp

```diff
@@ -0,0 +1,36 @@
+//Subsequence of a String using bitmasking
+#include<iostream>
+#include<string>
+using namespace std;
+void subsequence_bitmasking(string s)
+{
+    //Calculate length of string 
+    int len = s.length();
+
+    //Number of subsequence are always 1>>length
+    
+    for(int i=0;i<(1<<len);i++)
```

Comment:
Please add spaces around operator like `+`, `=` etc and space after operator like `;`.